Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -1256,7 +1258,34 @@ int CLuaEngineDefs::EngineRemoveShaderFromWorldTexture(lua_State* luaVM)
 
 uint CLuaEngineDefs::EngineGetModelTXDID(uint uiModelID)
 {
-    return g_pGame->GetRenderWare()->GetTXDIDForModelID(uiModelID);
+    CModelInfo* pModelInfo = g_pGame->GetModelInfo(uiModelID);
+
+    if (uiModelID >= 20000 || !pModelInfo)
```

Comment:
Don't use magic numbers [Use an existing function or something of similar nature]

Reply:
It is only draft =)
Will be changed